mbuild/tests/test_compound.py

```diff
@@ -619,12 +621,8 @@ def test_energy_minimization_ports(self, octane):
         assert np.array_equal(orientations, updated_orientations)
 
     def test_energy_minimize_openmm(self, octane):
-        import foyer
-        ff_xml = '/'.join(foyer.__file__.split('/')[:-1])+"/forcefields/oplsaa.xml"
+        ff_xml = get_fn('oplsaa.xml')
```

Comment:
Ah sorry I missed this question as well. I believe you can use the following pattern to get these files without adding them to mbuild:

https://github.com/mosdef-hub/foyer/blob/master/foyer/forcefield.py#L237